langgraph/graph/graph.py

```diff
@@ -106,18 +107,31 @@ def __init__(self) -> None:
     def _all_edges(self) -> set[tuple[str, str]]:
         return self.edges
 
-    def add_node(self, key: str, action: RunnableLike) -> None:
+    @overload
+    def add_node(self, node: RunnableLike) -> None:
+        ...
+
+    @overload
+    def add_node(self, node: str, action: RunnableLike) -> None:
```

Comment:
I guess the alternative of having the action be first and name second would be weird because of the current behavior

Reply:
yea I think that would be confusing to have both